libkbfs/backpressure_disk_limiter.go

```diff
@@ -1049,17 +1053,23 @@ func (bdl *backpressureDiskLimiter) reserveBytes(
 	}
 
 	bdl.overallByteTracker.updateFree(freeBytes)
-	count := bdl.overallByteTracker.tryReserve(blockBytes)
-	if count < 0 {
-		return count, nil
+	if typ != syncCacheLimitTrackerType {
+		count := bdl.overallByteTracker.tryReserve(blockBytes)
+		if count < 0 {
+			return count, nil
+		}
+		// We calculate the total free bytes by adding the reported free bytes and
+		// the non-`tracker` used bytes.
+		tracker.updateFree(freeBytes + bdl.overallByteTracker.used -
+			tracker.usedResources())
+	} else {
+		// TODO: this allows the sync cache to take up 100% of free space
```

Comment:
Is this really a "TODO"?  Seems like an explanatory comment.

Reply:
lol, that would be me spacing out when deleting a TODO and accidentally keeping the TODO part of the comment and simply replacing its contents with the following comment.